pkg/elf/elf_test.go

```diff
@@ -36,7 +34,7 @@ type ELFTestSuite struct{}
 var (
 	_ = Suite(&ELFTestSuite{})
 
-	baseObjPath = filepath.Join(testutils.CiliumRootDir, "test", "bpf", "elf-demo.o")
+	baseObjPath = filepath.Join("..", "..", "test", "bpf", "elf-demo.o")
```

Comment:
Same question as kkourt's above - will this work when run from within `pkg/elf`?

Reply:
Yes.